src/common/load-plugins.js

```diff
@@ -40,18 +46,19 @@ async function load(plugins, pluginSearchDirs) {
     (plugin) => typeof plugin === "string"
   );
 
-  const externalManualLoadPlugins = externalPluginNames.map((name) => {
-    let plugin;
-    try {
-      // try local files
-      plugin = requireFrom(path.resolve(name), process.cwd());
-    } catch {
-      // try node modules
-      plugin = requireFrom(name, process.cwd());
-    }
-
-    return { name, ...plugin };
-  });
+  const externalManualLoadPlugins = await Promise.all(
+    externalPluginNames.map(async (name) => {
+      try {
+        // try local files
+        const module = await import(pathToFileURL(path.resolve(name)).href);
+        const plugin = module.default ?? module;
+        return { name, ...plugin };
+      } catch {
+        // try node modules
+        return importPlugin(name, process.cwd());
```

Comment:
This line should be covered by tests..

Reply:
I'm 100% sure this line is covered, there is something wrong with c8, see  https://github.com/nicolo-ribaudo/jest-light-runner/pull/13